skuld/openstackmap.py

```diff
@@ -481,79 +485,128 @@ def _get_neutron_data(self):
 
     def load_nova(self):
         """load nova data: vms"""
-        if self.objects_strategy == OpenStackMap.USE_CACHE_OBJECTS and \
+        if (self.objects_strategy == OpenStackMap.USE_CACHE_OBJECTS or
```

Comment:
It is documented in the docstring of the constructor, when this parameter is set.
The USE_CACHE_OBJECTS_ONLY is the new value added. The difference is that USE_CACHE_OBJECTS use the cache values if available, otherwise it gets them from the server (an store them in the cache), while USE_CACHE_OBJECTS_ONLY never ask for the values to the server.  

That is, USE_CACHE_OBJECTS_ONLY is for offline use. Indeed, another change is that with USE_CACHE_OBJECTS before reading values from a server, there is an authentication to get the endpoints and check if this resource (e.g. neutron) is available in that region, while with USE_CACHE_OBJECTS_ONLY this is not needed, because the check is test the existence in the cache of information for this service and region.